resources/views/category/CreateCat.blade.php

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
make name small case, and same with functions cat is not good name for category. Because it can be misunderstood with other developers. Writing good names for files and functions or any where in your code, is important so please be keen about naming.